bdb/bdb_verify.c

```diff
@@ -815,7 +815,7 @@ static int bdb_verify_ll(
 
                         if (rc == 0) {
                             realblobsz[blobno] = dbt_blob_data.size;
-                            if (blobsizes[blobno] == -1 && rc != DB_NOTFOUND) {
```

Comment:
What was the warning here?

Reply:
`rc != DB_NOTFOUND` will always to true as `if (rc == 0)` guarantees it.

Reply:
right